mesos-commons/src/main/java/org/apache/mesos/collections/Maps.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.collections;
+
+import com.google.common.base.Predicate;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ */
+public class Maps {
+
+  public static Map<String, String> filterMapKeysThatStartWith(Map<String, String> map, final String startWithString) {
+    return com.google.common.collect.Maps.filterKeys(map, new Predicate<String>() {
```

Comment:
I would probably just rename this method filterMapKeys for simplicity, but this is just a preference.